tests/frameworks/gtest/include/gtest/gtest-param-test.h

```diff
@@ -1406,6 +1406,8 @@ internal::CartesianProductHolder10<Generator1, Generator2, Generator3,
       GTEST_TEST_CLASS_NAME_(test_case_name, test_name)::AddToRegistry(); \
   void GTEST_TEST_CLASS_NAME_(test_case_name, test_name)::TestBody()
 
+# define TEST_P(test_case_name, test_name) GTEST_TEST_P_(test_case_name, test_name, test_case_name)
```

Comment:
>TEST_P [](start = 9, length = 6)

Just so I understand this, you are modifying the gtest api to have another level of indirection here so we can do the interstitial doodad?